.github/workflows/publish-chart.yaml

```diff
@@ -18,7 +23,9 @@ env:
   KIND_VERSION: "v0.14.0"
   KIND_IMAGE: "kindest/node:v1.23.6@sha256:b1fa224cc6c7ff32455e0b1fd9cbfd3d3bc87ecaa8fcb06961ed1afb3db0f9ae"
 jobs:
+  # `test-chart` job starts if a PR with Helm Chart is created, merged etc.
   test-chart:
+    if: github.event_name != 'push'
```

Comment:
I don't think we need to start testing if the PR has been merged.

Reply:
I thought that yes.
Trivy Helm chart depends on Trivy image with a specific tag. If this image is removed, we shouldn't publish a new chart.
Does it wrong point?

Reply:
hm... i forgot about this case.
Thanks for reminding me!